<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Вход в приложение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login_line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="Enter_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Продолжить</string>
    </property>
   </widget>
   <widget class="QLabel" name="Login_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Password_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_login_line_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Это поле не должно быть       пустым</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Error_PassWord_line_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Это поле не должно быть пустым</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Нет аккаунта?</string>
    </property>
   </widget>
   <widget class="QLabel" name="Registration_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарегестрироваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="View_PassWord_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>226</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Leave_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(237, 51, 59);
color: black;
</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
